### DISCUSSION TITLE
Export UV only

### DISCUSSION THREAD

**Discussion by mdubrois** [Autodesk/maya-usd#1780] [Q&A] [answered]
Hello,

Is there a way to export the UVs only like it's done in the AL plugin?
Being able to determine which kind of content should be exported will be very useful.
In my case, I would like to be able to overrride the UVs made by the modeling with the UVs made by the surfacing step.

Cheers,

Marc Dubrois

**Comment by kxl-adsk** [comment, marked as answer]
Hi Mac,

No, we don't currently have such granular control over the export. One way to work around this limitation would be to export to a layer in memory and then copy the bits of information you need to a new layer that you will save.

Cheers,
KxL

**Reply by mdubrois** [reply]
Ok, thank you Krystian, I will do that.
